elastic/logstash/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM openjdk:8-jdk-alpine
+
+# Add packages
+RUN apk add --update --no-cache bash wget make java-cacerts go git musl-dev openssl
+
+# Install env2yaml
+RUN mkdir -p /app/golang
+WORKDIR /app/golang
+COPY bin/env2yaml.go /app/golang
+RUN go get gopkg.in/yaml.v2
+RUN go build -o env2yaml
+RUN cp /app/golang/env2yaml /usr/local/bin
+
+
+# Add Logstash.
+WORKDIR /app
+RUN wget -q https://artifacts.elastic.co/downloads/logstash/logstash-6.7.0.tar.gz
+RUN tar -zxf logstash-6.7.0.tar.gz -C /app
+RUN mv /app/logstash-6.7.0 /app/logstash
+
+ENV ELASTIC_CONTAINER false
+ENV PATH=/app/logstash/bin:$PATH
+
+# Add logstash config files
+COPY dictionaries /app/logstash/dictionaries
+COPY pipeline/ /app/logstash/pipeline/
+COPY config/ /app/logstash/config/
+
+# Ensure Logstash gets a UTF-8 locale by default.
+ENV LANG='en_US.UTF-8' LC_ALL='en_US.UTF-8'
+
+# Update permissions to be accessible by randomly generated user ID
```

Comment:
What User Id is randomly generated?

Reply:
We never know. Openshift basically generated a random ID like `10002342` and it can be different for each pod and each envt.